draft-ietf-quic-tls.md

```diff
@@ -316,48 +316,48 @@ same keys even if TLS has already updated to newer keys.
 
 One important difference between TLS records (used with TCP) and QUIC CRYPTO
 frames is that in QUIC multiple frames may appear in the same QUIC packet as
-long as they are associated with the same encryption level. For instance, an
-implementation might bundle a Handshake message and an ACK for some Handshake
-data into the same packet.
+long as they are associated with the same packet number space. For instance,
+an endpoint can bundle a Handshake message and an ACK for some Handshake data
+into the same packet.
 
-Some frames are prohibited in different encryption levels, others cannot be
-sent. The rules here generalize those of TLS, in that frames associated with
-establishing the connection can usually appear at any encryption level, whereas
-those associated with transferring data can only appear in the 0-RTT and 1-RTT
-encryption levels:
+Some frames are prohibited in different packet number spaces. The rules here
+generalize those of TLS, in that frames associated with establishing the
+connection can usually appear in packets in any packet number space, whereas
+those associated with transferring data can only appear in the application
+packet number space:
```

Comment:
Looking at [section 12.3 of the transport draft](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#section-12.3), I think the term we use is "application data (packet number) space"?

This comment applies to "application packet number space" used elsewhere in this PR too.